### DISCUSSION TITLE
Heroku App Webhooks

### DISCUSSION THREAD

**Discussion by Trimud** [vercel/next.js#19407] [Help]
Hello,

I'm trying to integrate [App Webhooks](https://devcenter.heroku.com/articles/app-webhooks-tutorial) from Heroku and notify my users that there is a new build of the app. Possibly force them to refresh the page.
I struggle to wrap my head around how to do this - probably via /app endpoint which will listen for updates but how do I notify the frontend for this? Should I do some polling or else?
Another possible way is to use https://www.npmjs.com/package/eventsource.

Any suggestions will be highly appreciated.
Thanks in advance!

**Comment by hdgone** [comment]
I'm not sure I understand your use case correctly. Do you want to notify your users that are currently on the website that you have just deployed the new version of the app?

**Reply by Trimud** [reply]
Yes, and ask them to refresh the page to get the new version.
